packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -544,6 +544,10 @@ export class DocumentDeltaConnection
         if (typeof error !== "object") {
             message = `${message}: ${error}`;
         } else if (error?.type === "TransportError") {
+            // JSON.stringify drops Error.message
```

Comment:
#nit We're already using a stringify replacer to serialize this error. What is your take on using it to include any error object properties explicitly in there and keep the message in the same format?

Reply:
I'm pretty sure that the reason they are not showing up in JSON.stringify() flow is because these properties are not enumerable. And thus there is nothing callback provided to JSON.stringify can do to make them show up in the list.
So it can't really include it, but we can wrap this process (of getting error.message & dealing with circular references) into helper function for sure. I'll mention it in https://github.com/microsoft/FluidFramework/issues/8165